app/services/idv/vendor_result.rb

```diff
@@ -1,6 +1,12 @@
 module Idv
   class VendorResult
-    attr_reader :success, :errors, :messages, :normalized_applicant, :timed_out, :exception
+    attr_reader :success,
```

Comment:
added `context` so this doesn't break if someone uses it in the proofer response.